libs/core/src/lib/carousel/carousel.component.scss

```diff
@@ -4,6 +4,26 @@ fd-carousel:focus {
  outline: 1px dotted;
 }
 
+.fd-carousel:hover {
+  .fd-carousel__content {
+    .fd-carousel__button {
+      &.fd-button {
+        display: block;
+      }
+    }
+  }
+}
+
+.fd-carousel:hover {
+  .fd-carousel__content {
+    .fd-carousel__button {
+      &.fd-button {
+        display: block;
+      }
+    }
+  }
+}
```

Comment:
This should be in fundamental-styles

Reply:
have raised https://github.com/SAP/fundamental-styles/pull/2431 in styles.